tests/Flags.test.ts

```diff
@@ -0,0 +1,89 @@
+import { Lexer, Parser } from 'lexure';
+import { FlagStrategy } from '../src/lib/utils/strategies/FlagUnorderedStrategy';
+
+const parse = (testString: string) => {
+	return new Parser(
+		new Lexer(testString)
+			.setQuotes([
+				['"', '"'],
+				['“', '”'],
+				['「', '」']
+			])
+			.lex()
+	)
+		.setUnorderedStrategy(new FlagStrategy(['f', 'hello']))
+		.parse();
+};
+
+describe('Flag parsing strategy', () => {
+	test('GIVEN typeof FlagStrategy RETURNS function', () => {
+		expect(typeof FlagStrategy).toBe('function');
+	});
+	test('GIVEN signle-hypen flag RETURNS flag', () => {
```

Comment:
Typo, `single`, this applies to all btw.

Reply:
I'll fix this later as it's very minor

Reply:
As long as you fix it before the PR is merged